crowd_anki/config/config_dialog.py

```diff
@@ -0,0 +1,75 @@
+"""
+Config dialog entry point.
+This module will load the config initially.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+
+from aqt.qt import *
+
+from .config_ui import Ui_Dialog as ConfigUI
+from .config_settings import ConfigSettings
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+class ConfigDialog(QDialog):
+    def __init__(self, parent=None):
+        super().__init__()
+        self.parent = parent
+        self.config = ConfigSettings()
+        self.form = ConfigUI()
+        self.form.setupUi(self)
+        self.ui_initial_setup()
+
+    def accept(self):
+        invalid_values = self.config.find_invalid_config_values()
+        if invalid_values:
+            print(invalid_values)
+            self.error_message_popup = QMessageBox(self)
+            message_text = "\" , \"".join(invalid_values)
+            self.error_message_popup.setText(f"Invalid config values: \"{message_text}\"")
+            self.error_message_popup.open()
+        else:
+            self.config.set_all_values()
```

Comment:
this seems like an internal detail that should be called from `write_values_to_anki`